### PR TITLE
Patch VWE - Frontier Gauss and Laser addons.

### PR DESCRIPTION
**Pull request by N7Huntsman**
Patches the addon frontier weapons added when Gauss and Lasers are active. The weapons are ultra tech level, so they just use the existing ammo items.

### CODE REVIEW

**Comment by Breadbox2k19** [comment]
Can you take a look at patching the heavy laser rifle from UNSC armoury?

**Comment by N7Huntsman** [comment]
> Can you take a look at patching the heavy laser rifle from UNSC armoury?

Done. It's in #723

File: Patches/Vanilla Weapons Expanded - Frontier/Weapons_Frontier-Laser.xml [hunk 1]
```diff
@@ -2,28 +2,206 @@
...
+        </FireModes>
+        <weaponTags>
+          <li>SalvagedLaserGun</li>
+        </weaponTags>		
+      </li>
+
+      <!-- === Salvaged Laser Hunting Rifle === -->
+
+      <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+        <defName>VWEFT_Bullet_SalvagedLaserHuntingRifle</defName>
```
Comment (Tostov) [inline Patches/Vanilla Weapons Expanded - Frontier/Weapons_Frontier-Laser.xml:171]: Patch is targeting the bullet, not the weapon itself.

**Review by Tostov — CHANGES_REQUESTED** [review]
Seems to be fine outside of mistargeted patch for the laser hunting rifle

File: Patches/Vanilla Weapons Expanded - Frontier/Weapons_Frontier-Laser.xml (hunk 1 above)
Comment (N7Huntsman) [reply Patches/Vanilla Weapons Expanded - Frontier/Weapons_Frontier-Laser.xml:171]: Opps, got sloppy with a find+replace. Thanks.

**Review by Tostov — APPROVED** [review]
All looks good to go now.

**Comment by Tostov** [comment]
Not big enough to warrant change request, but I'd advise changing the description of the ammodef to include the cartridge used, for example for the gauss revolver so people don't have to go into the info menu to check.

**Pull request merged**